chemprop/data/data.py

```diff
@@ -175,6 +188,24 @@ def number_of_molecules(self) -> int:
         """
         return len(self.smiles)
 
+    @property
+    def number_of_atoms(self) -> int:
+        """
+        Gets the number of atoms in the :class:`MoleculeDatapoint`.
+
+        :return: The number of atoms.
+        """
+        return len(self.mol[0].GetAtoms())
+
+    @property
+    def number_of_bonds(self) -> int:
+        """
+        Gets the number of bonds in the :class:`MoleculeDatapoint`.
+
+        :return: The number of bonds.
+        """
+        return len(self.mol[0].GetBonds())
+
```

Comment:
This is a good addition. We should have added these properties a long time ago.

Reply:
Thanks! However, it only returns the value of the first molecule (`--number_of_molecules` is 1). I should modify this to return an array containing the values for each molecule (`--number_of_molecules` > 1).